src/ITILValidationTemplate.php

```diff
@@ -0,0 +1,174 @@
+{
+    // From CommonDBTM
+    public $dohistory = true;
+
+    public static $rightname = 'itilvalidationtemplate';
+
+    public $can_be_translated = true;
+
+    public static function getTypeName($nb = 0)
+    {
+        return _n('Validation template', 'Validation templates', $nb);
+    }
+
+    public function getAdditionalFields()
+    {
+        return [
+            [
+                'name'  => 'approver',
+                'label' => __('Approver'),
+                'type'  => '',
+                'list'  => true
+            ],
+            [
+                'name'           => 'content',
+                'label'          => __('Content'),
+                'type'           => 'tinymce',
+                // As content copying from template is not using the image pasting process, these images
+                // are not correctly processed. Indeed, document item corresponding to the destination item is not created and
+                // image src is not containing the ITIL item foreign key, so image will not be visible for helpdesk profiles.
+                // As fixing it is really complex (requires lot of refactoring in images handling, both on JS and PHP side),
+                // it is preferable to disable usage of images in templates.
+                'disable_images' => true,
+            ]
+        ];
+    }
+
+    /**
+     * Display specific "approver" field
+     *
+     * @param $value
+     * @param $name
+     * @param $options
+     */
+    public static function displayValidatorField(
+        $value = null,
+        $options = []
+    ) {
+        if ($value) {
+            $itemtype = array_key_first($value);
+            $items_id = array_map(function ($item) {
+                return $item['items_id'];
+            }, $value[$itemtype]);
+        }
+
+        $options['users_id_requester'] = \Session::getLoginUserID();
+        $options['itemtype_target']    = $itemtype ?? null;
+        $options['groups_id']          = $value ? $value[$itemtype][0]['groups_id'] : null;
+        $options['items_id_target']    = $value && $itemtype == 'Group' && count($items_id) == 1 ?
+            $items_id[0] : $items_id ?? null;
+        $options['right']              = 'validate_request';
+        $options['display']            = false;
+
+        return CommonITILValidation::dropdownValidator($options);
+    }
+
+    public function displaySpecificTypeField($id, $field = [], array $options = [])
+    {
+        if ($field['name'] == 'approver') {
+            echo self::displayValidatorField(ITILValidationTemplate_Target::getTargets($id), []);
+        }
+    }
+
+    public static function getSpecificValueToSelect($field, $name = '', $values = '', array $options = [])
+    {
+        if ($field == 'approver') {
+            return self::displayValidatorField($values[$field], $options);
+        }
+
+        return parent::getSpecificValueToSelect($field, $name, $values, $options);
+    }
```

Comment:
IMHO, call to `displayValidatorField` would not work as `$values[$field]` would not have the expected format. Anyway, there is no search option for the `approver` field. Unless you are going to add a search option to display approvers list, I think that this code is not reachable and can be removed.